src/vl-mapactions-all.js

```diff
@@ -96,4 +97,5 @@ export {
   OlExtent,
   OlLoadingstrategy,
   OlFullScreenControl,
+  proj4,
```

Comment:
Vraag: wat is de motivatie/voordeel achter het hier exporteren en in de vl-map importeren vanuit vl-mapactions. I.p.v. rechtstreeks importeren in vl-map van proj4?
Is de redenering dat je dit enkel expliciet in dependencies van vl-mapactions wil zetten?

Reply:
Probleem is dat proj4 door zijn huidige setup native niet rechtstreeks geïmporteerd kan worden in de vl-map. Ik heb gezocht naar native varianten, maar kon niet meteen iets vinden dat voldoende kwaliteit garandeert.

Er zou hiervoor dus een build tool geïntroduceerd moeten worden in vl-map die potentieel conflicten gaat geven met de huidige setup. De vl-mapactions bevat reeds Webpack aangezien OpenLayers verwerkt moest worden in de export. Het leek me interessant om op die plaats proj4 ook te voorzien als export aangezien het een dependency is die zeer vaak nodig zal zijn in combinatie met vl-mapactions rechtstreeks of onrechtstreeks bij gebruik van vl-map.

Een alternatief zou kunnen zijn dat we een nieuwe repo voorzien en proj4 zelf aanbieden als ES build zodat het rechtstreeks geïmporteerd kan worden bij de vl-map. Dat leek me overkill t.o.v. de huidige oplossing.

Reply:
OK, got it. Ik kon het niet onmiddellijk afleiden uit de initiële context.

Een aparte repo is idd overkill. Misschien moet er issue bij proj4 gelogd worden indien het probleem effectief daar ligt...